### ISSUE TITLE
print(AllHelp)

### ISSUE THREAD

**Issue by joesh1** [Qirky/FoxDot#107]
Hi Ryan, one thing that would be so helpful would be a simple command print(HelpAll), or print(AllAttributes) or something which prints out in the post window all the various print(SynthDefs), print(BufferManager()), print(Clock.sleep_time), print(Player.get_attributes()), print(FxList) and so on. Every time I'm looking for something I need to check back on earlier code or examples to remember some of these. One command to print all these in a list on the post window would be very helpful?

**Comment by Qirky** [comment]
Good idea! I'll add it to the help message (`Ctrl+H`) and a menu option to display the message in the console!

**Comment by joesh1** [comment]
Hi, I haven't checked yet, but I assume it's not in there yet - I just checked and saw only print(SynthDefs) and print(Samples). I'm on Mac so you mean in the (`cmd+K`) section.
